content/en/docs/distributions/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -182,51 +318,116 @@ custom providers.
+
+2. Update the `distribution/stacks/ibm/application/oidc-authservice-appid/parms.env` and `distribution/stacks/ibm/application/oidc-authservice-appid/secret_params.env` with values collected in [Prereq](#prerequisites-1) section. If at any point the values change and you have to change them, you can either patch the [configmap](#patch-configmap) and [secret](#patch-secret) or change the content in the files and apply the kustomize again. You will need to restart authservice by deleting the pod `kubectl delete po -n istio-system authservice-0` .
+
+To reapply the authservice only
+
+```bash
+kustomize build distributions/stacks/ibm/application/oidc-authservice-appid | kubectl apply -f -
+```
+
+3. Populate the `kustomization.yaml` file with the following.
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+# Cert-Manager (required)
+- ../distributions/stacks/ibm/application/cert-manager-kube-system-resources
+- ../distributions/stacks/ibm/application/cert-manager-crds
+- ../distributions/stacks/ibm/application/cert-manager
+# Istio (requires)
+- ../distributions/stacks/ibm/application/istio-1-9-0
+# OIDC Authservice (required)
+- ../distributions/stacks/ibm/application/oidc-authservice-appid
+# KNative
+- ../distributions/stacks/ibm/application/knative
+# Kubeflow namespace (required)
+- ../distributions/stacks/ibm/application/kubeflow-namespace
+# Kubeflow Roles (required)
+- ../distributions/stacks/ibm/application/kubeflow-roles
+# Kubeflow istio resource (required)
+- ../distributions/stacks/ibm/application/kubeflow-istio-resources
+# Kubeflow Pipelines Argo
+# - ../distributions/stacks/ibm/application/kfp-argo
+# Kubeflow Pipelines Tekton
+- ../distributions/stacks/ibm/application/kfp-tekton
+# KFServing
+- ../distributions/stacks/ibm/application/kfserving
+# Katib
+- ../distributions/stacks/ibm/application/katib
+# Central Dashboard (required)
+- ../distributions/stacks/ibm/application/centraldashboard/multi-user
+# Admission Webhook (required)
+- ../distributions/stacks/ibm/application/admission-webhook
+# Notebook Controller
+- ../distributions/stacks/ibm/application/notebook-controller
+# Jupyter Web App
+- ../distributions/stacks/ibm/application/jupyter-web-app/insecure
+# Profiles + KFAM (required)
+- ../distributions/stacks/ibm/application/profiles
+# Volumes Web App
+- ../distributions/stacks/ibm/application/volumes-web-app/insecure
+# Tensorboards Web App
+-  ../distributions/stacks/ibm/application/tensorboard-controller
+# Tensorboard Controller
+-  ../distributions/stacks/ibm/application/tensorboard-web-app/insecure
+# TFJob Operator
+- ../distributions/stacks/ibm/application/tf-job
+# Pytorch Operator
+- ../distributions/stacks/ibm/application/pytorch-job
+# MPI Operator
+- ../distributions/stacks/ibm/application/mpi-job
+# MXNet Operator
+- ../distributions/stacks/ibm/application/mxnet-job
+# XGBoost Operator
+- ../distributions/stacks/ibm/application/xgboost-job
+```
```

Comment:
I suggest keeping this `kustomization.yaml` from doc into this folder: https://github.com/kubeflow/manifests/tree/master/distributions/stacks/ibm